src/lighteval/main_accelerate.py

```diff
@@ -20,63 +20,187 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+import logging
 import os
-from datetime import timedelta
+from typing import Optional
 
-from lighteval.logging.evaluation_tracker import EvaluationTracker
-from lighteval.logging.hierarchical_logger import hlog_warn, htrack
-from lighteval.models.model_config import create_model_config
-from lighteval.pipeline import EnvConfig, ParallelismManager, Pipeline, PipelineParameters
-from lighteval.utils.imports import is_accelerate_available, is_tgi_available
+from typer import Argument, Option
+from typing_extensions import Annotated
 
 
-if not is_accelerate_available() and not is_tgi_available():
-    hlog_warn("Using either accelerate or text-generation to run this script is advised.")
+logger = logging.getLogger(__name__)
 
 TOKEN = os.getenv("HF_TOKEN")
-
-if is_accelerate_available():
+CACHE_DIR: str = os.getenv("HF_HOME", "/scratch")
+
+HELP_PANNEL_NAME_1 = "Common Paramaters"
+HELP_PANNEL_NAME_2 = "Logging Parameters"
+HELP_PANNEL_NAME_3 = "Debug Paramaters"
+HELP_PANNEL_NAME_4 = "Modeling Paramaters"
+
+
+def accelerate(  # noqa C901
+    # === general ===
```

Comment:
For the future, I wonder if we can extract those directly from the config files, to avoid duplicating doc

Reply:
for now the doc is duplicated but some argument don't make sense for endpoint for example so I think it's nice to have them completelty separated